tests/generic_config_updater/test_portchannel_interface.py

```diff
@@ -78,62 +78,7 @@ def setup_env(duthosts, rand_one_dut_hostname, portchannel_table):
         delete_checkpoint(duthost)
 
 
-def test_portchannel_interface_tc1_add_new_portchannel(duthost):
```

Comment:
>test_portchannel_interface_tc1_add_new_portchannel [](http://example.com/codeflow?start=4&length=50)

why delete this testcase?

Reply:
The purpose of this test is to verify portchannel interface. To verify the case of adding portchannel interface, it could be done in [portchannel_interface_tc1_add_and_rm](https://github.com/Azure/sonic-mgmt/pull/5866/files/43a79d26c30b5130e284feccd01ea3fd10c88d29#diff-0423d7a8e21710875b65da2c40f8589ac04d0a108345a1700ec1eac646f2f127R155)